bootstrap/bootstrap.go

```diff
@@ -209,47 +209,75 @@ func (b *Bootstrap) Cancel() error {
 	return nil
 }
 
+type HookConfig struct {
+	Name           string
+	Scope          string
+	Path           string
+	Env            env.Environment
+	SpanAttributes map[string]string
+}
+
+func (b *Bootstrap) tracingImplementationSpecificHookScope(scope string) string {
```

Comment:
There's probably a backlog TODO in here to rename the hook scopes (or “locations”?) throughout the codebase so that they match the documented names.

Reply:
🤔 i wonder if we should just do it now (or soon, in another, less-dense PR). I don't thiiiiink it would break anything?

Reply:
Yeah I think “now”, but probably the “sleevie” version of now 😉
Definitely separate pull request, and probably easier to get this one merged first.